index.js

```diff
@@ -116,6 +122,9 @@ module.exports = fp(async function (app, opts) {
         for (const prop of Object.keys(resolver)) {
           type[prop] = resolver[prop]
         }
+      } else if (type instanceof GraphQLInterfaceType || type instanceof GraphQLUnionType) {
+        const resolver = resolvers[name]
+        type.resolveType = resolver.__resolveType
```

Comment:
Is this common syntax in the graphql world?

Reply:
You are right this is not part of the official `graphql` lib, I confused with Apollo's makeExecutableSchema which use `__` syntax. 

I was looking for a way to add union and interface without using `graphql-tool` or raw schema definition. But I've mixed both libraries ... 👎 

I will remove the `__` to match the official `graphql` library, does it suits you ? 
Adding `isTypeOf` and `resolveType` will allow users to define resolvers for union and interface defined in the schema definition.

Reply:
Awesome!